specification/monitor/resource-manager/Microsoft.Insights/preview/2020-01-01-preview/examples/createOrUpdateManagementGroupDiagnosticSetting.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "managementGroupId": "testChildMG7",
+    "name": "setting1",
+    "api-version": "2020-01-01-preview",
+    "parameters": {
+      "properties": {
+        "storageAccountId": "/subscriptions/bfaef57f-297e-4210-bfe5-27c18cc671f7/resourceGroups/FuncAppRunners/providers/Microsoft.Storage/storageAccounts/testpersonalb6a5",
+        "workspaceId": "/subscriptions/9cf7cc0a-0ba1-4624-bc82-97e1ee25dc45/resourceGroups/mgTest/providers/Microsoft.OperationalInsights/workspaces/mgTestWorkspace",
+        "eventHubAuthorizationRuleId": "/subscriptions/fb9f25f9-5785-4510-a38f-a62f188eb9f8/resourceGroups/montest/providers/microsoft.eventhub/namespaces/mynamespace/eventhubs/myeventhub/authorizationrules/myrule",
+        "eventHubName": "myeventhub",
+        "logs": [
+          {
+            "category": "Administrative",
+            "enabled": true
+          },
+          {
+            "category": "Policy",
+            "enabled": true
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "providers/Microsoft.Management/managementGroups/testChildMG7/providers/microsoft.insights/diagnosticSettings/setting1",
+        "type": null,
```

Comment:
type should not be null in examples

Reply:
@pilor what should it be then?
This is the output I get when I test the API in Postman

Reply:
If your API is outputting "Type" as null it needs to be fixed.  This is your resource type "Microsoft.Insights/diagnosticSettings".

Reply:
Made the change here. Will also update the API to emit it correctly